packages/bpk-component-calendar/src/BpkCalendarGrid.js

```diff
@@ -139,8 +137,6 @@ class BpkCalendarGrid extends Component {
               markToday={markToday}
               markOutsideDays={markOutsideDays}
               selectionConfiguration={selectionConfiguration}
-              selectionEnd={selectionEnd}
-              selectionStart={selectionStart}
```

Comment:
Is it right these got removed even though they aren't marked as deprecated?

Reply:
Yes, I think it was just something we accidentally forgot, but we did add it to the Changelog - https://github.com/Skyscanner/backpack/blob/main/CHANGELOG.md#2021-11-15
They're also not actually used anywhere in the Week component anyways so it has no effect!